arangod/Aql/SkipResult.cpp

```diff
@@ -40,14 +40,19 @@ auto SkipResult::didSkip(size_t skipped) -> void {
   _skipped.back() += skipped;
 }
 
-auto SkipResult::didSkipSubquery(size_t skipped, size_t depth) -> void {
+template<int depthOffset>
+requires(depthOffset == 0 || depthOffset == -1)  //
```

Comment:
let's see if all compilers used for 3.11 support this yet...

Reply:
I guess so, we've had e.g. this in for a while: https://github.com/arangodb/arangodb/pull/17644/files#diff-d516d61017cef8d3f4ba883997c93b18e838008f49bd03add75d5ce0caa44cf4